lumicks/pylake/kymotracker/detail/peakfinding.py

```diff
@@ -31,10 +32,15 @@ def find_kymograph_peaks(
         (time, position) coordinates. Default: no cropping.
     filter_width : float
         Width of the point spread function in pixels. Default: 0.5.
+    adjacency_half_width : int, optional
+        When provided, a detection always needs a detection within a certain cutoff radius (in
+        pixels) in an adjacent frame. This can be used to suppress singular noise peaks.
```

Comment:
nit: A small numeric example with a 3x3 matrix might be nice to have, for the docstring.

Reply:
nice example!